src/Symfony/Component/Messenger/Command/AbstractFailedMessagesCommand.php

```diff
@@ -175,7 +175,7 @@ protected function printPendingMessagesMessage(ReceiverInterface $receiver, Symf
     protected function getReceiver(/* string $name */): ReceiverInterface
     {
         if (1 > \func_num_args() && __CLASS__ !== static::class && __CLASS__ !== (new \ReflectionMethod($this, __FUNCTION__))->getDeclaringClass()->getName() && !$this instanceof \PHPUnit\Framework\MockObject\MockObject && !$this instanceof \Prophecy\Prophecy\ProphecySubjectInterface) {
-            @trigger_error(sprintf('The "%s()" method will have a new "string $name" argument in version 5.3, not defining it is deprecated since Symfony 5.2.', __METHOD__), \E_USER_DEPRECATED);
+            trigger_deprecation('symfony/messenger', '5.2', 'The "%s()" method will have a new "string $name" argument in version 5.3, not defining it is deprecated.', __METHOD__);
```

Comment:
I know, this is not related to this PR, but I don't understand the message mixing 5.2 and 5.3

Reply:
This is from #38468

Reply:
Fixed in #40667